src/ceph-daemon/ceph-daemon

```diff
@@ -1095,6 +1095,14 @@ def command_bootstrap():
         ], {
             tmp_key.name: '/tmp/key:z',
         })
+        cli([
+            'config-key',
+            'set',
+            'mgr/ssh/ssh_identity_pub',
+            '-i', '/tmp/pub'
```

Comment:
whos creating that file?

Reply:
see the next line.. it's passed into the container from the tmp_pub temp file